exam-trial-2/src/main/java/org/example/CsvWriter.java

```diff
@@ -0,0 +1,21 @@
+package org.example;
+
+import com.opencsv.bean.StatefulBeanToCsv;
+import com.opencsv.bean.StatefulBeanToCsvBuilder;
+import com.opencsv.exceptions.CsvDataTypeMismatchException;
+import com.opencsv.exceptions.CsvRequiredFieldEmptyException;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.Writer;
+import java.nio.file.Path;
+import java.util.List;
+
+public class CsvWriter {
```

Comment:
I mean yes, but you could have used a common interface for your two writers and you could have used Strategy pattern here

the logic itself looks ok